package.json

```diff
@@ -26,6 +26,7 @@
     "test:server-protocols": "yarn build:server-protocols && lerna run test --scope '@aws-sdk/*-server'",
     "pretest:e2e": "yarn build:crypto-dependencies && lerna run --scope '@aws-sdk/{client-cloudformation,karma-credential-loader}' --include-dependencies build",
     "test:e2e": "node ./tests/e2e/index.js",
+    "test:versions": "ts-node tests/versions/tslib.ts",
```

Comment:
Should `test:versions` be called on `test:all`, before unit tests are run?

Reply:
It's already included if I use`*.spec.ts` to name the file